pgcli/key_bindings.py

```diff
@@ -25,12 +30,15 @@ def _(event):
         Enable/Disable Multiline Mode.
         """
         _logger.debug('Detected F3 key.')
-        line.always_multiline = not line.always_multiline
+        buf = event.cli.current_buffer
+        buf.always_multiline = not buf.always_multiline
 
-    @handle(Keys.ControlSpace, in_mode=InputMode.INSERT)
+    @handle(Keys.ControlSpace, filter=~filters.HasSelection())
```

Comment:
Not 100% confident that this is the same thing, but casual testing seemed to indicate so.